setup1-cli-tools/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "setup1-cli-tools"
+version = "0.1.0"
+authors = ["The Aleo Team <hello@aleo.org>"]
+edition = "2018"
+
```

Comment:
perhaps add a `description` field, or a README.md explaining more the purpose of these tools?

Reply:
Let's add a PR for CLI tools documentation

Reply:
yep, I'd prefer to write a README.md, either here or in a new PR, depending how faster are we going to merge this one